docs/build-scripts.md

```diff
@@ -0,0 +1,20 @@
+## Build scripts
```

Comment:
We should make it clear that this folder is all about building the `dist` folder. It has a a few different steps and they are all launched from `sync.sh`.

This means putting the sync.sh section at the top could make some sense.